bluepyopt/ephys/parameters.py

```diff
@@ -228,7 +228,7 @@ def instantiate(self, sim=None, icell=None):
 
         for location in self.locations:
             for isection in location.instantiate(sim=sim, icell=icell):
-                for seg in isection:
+                for seg in isection.allseg():
```

Comment:
What exactly the difference here ? 
'for seg in section' is normally the advised way of doing this